src/VisualStudio/Core/Def/RoslynPackage.cs

```diff
@@ -218,17 +218,14 @@ private async Task LoadComponentsBackgroundAsync(CancellationToken cancellationT
             }
         }
 
-        private async Task LoadInteractiveMenusAsync(CancellationToken cancellationToken)
+        private async Task LoadInteractiveMenusAsync(CancellationToken _)
```

Comment:
have asked for an overload of GetServiceAsync that takes a CT so that we can still bail out after the switch to the UI thread if things were canceled.

Reply:
I prefer to keep the parameter with the original name either way.